operators/admin/olm-managing-custom-catalogs.adoc

```diff
@@ -34,10 +38,17 @@ include::modules/olm-mirroring-package-manifest-catalog.adoc[leveloffset=+2]
 .Additional resources
 
 * xref:../../operators/operator_sdk/osdk-generating-csvs.adoc#olm-arch-os-support_osdk-generating-csvs[Architecture and operating system support for Operators]
+* If your catalog image is hosted on a private registry and requires authentication, see xref:../../operators/admin/olm-managing-custom-catalogs.adoc#olm-accessing-images-private-registries_olm-managing-custom-catalogs[Adding pull secrets to catalog sources].
 
 include::modules/olm-updating-operator-catalog-image.adoc[leveloffset=+2]
 .Additional resources
 
 * xref:../../operators/operator_sdk/osdk-generating-csvs.adoc#olm-arch-os-support_osdk-generating-csvs[Architecture and operating system support for Operators]
 
 include::modules/olm-testing-operator-catalog-image.adoc[leveloffset=+2]
+
+include::modules/olm-accessing-images-private-registries.adoc[leveloffset=+1]
+.Additional resources
+
+* See xref:../../builds/creating-build-inputs.adoc#builds-secrets-overview_creating-build-inputs[What is a secret?] for more information on the types of secrets, including those used for registry credentials.
+* See xref:../../openshift_images/managing_images/using-image-pull-secrets.html#images-update-global-pull-secret_using-image-pull-secrets[Using image pull secrets] for more details on the impact of updating the global cluster pull secret.
```

Comment:
The link text here refers to the page but the link is to the section. Should the text be the section title instead?

Reply:
I went with the assembly title cuz I thought it was a little more useful in this context, vs saying something like "See 'Updating the global cluster pull secret' for more information on updating the global cluster pull secret." :shrug:

Reply:
Ended up going with:

```
* See xref:../../openshift_images/managing_images/using-image-pull-secrets.html#images-update-global-pull-secret_using-image-pull-secrets[Updating the global cluster pull secret] for more details on the impact of changing this secret.
```